pgmpy/models/MarkovChain.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+import numpy as np
+import networkx as nx
+
+from pgmpy.utils import sample_discrete
+
+
+class MarkovChainMonteCarlo(nx.DiGraph):
```

Comment:
@pratyakshs Since this class represents a Markov chain. The name should be Markov Chain.